### PR TITLE
DOC Add session-manager documentation

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/developer-docs/issues/322

### CODE REVIEW

File: en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+After:
+  - '#session-manager-rememberloginhash'
+---
+SilverStripe\Security\RememberLoginHash:
+  logout_across_devices: true
+```
+
+Please note, this configuration only removes "remember me" tokens on logout, it does not terminate active sessions across devices on logout.
+
+Read [Saved User Logins](https://docs.silverstripe.org/en/developer_guides/security/member/#saved-user-logins) to learn how to configure the "Keep me signed in" feature for your members.
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md]:
```suggestion
Read [Saved User Logins](/developer_guides/security/member/#saved-user-logins) to learn how to configure the "Keep me signed in" feature for your members.
```

File: en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+of the user activating the task, so if this is triggered via the browser, that user
+will need to log back in to perform further actions.
+
+## Caveats
+
+ * Every request with a logged-in member causes a database write (updating [`LoginSession`](api:SilverStripe\SessionManager\Models\LoginSession)), potentially affecting performance
+ * Restoring a database from an older snapshot will invalidate current sessions.
+ * PHP sessions can become out of sync with [`LoginSession`](api:SilverStripe\SessionManager\Models\LoginSession) objects. Both can exist beyond their expiry date.
+   This is not an issue in practice since the association between the two is checked on each session-based request
+   (through [`LoginSessionMiddleware`](api:(api:SilverStripe\SessionManager\Middleware\LoginSessionMiddleware)).
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md]:
```suggestion
   (through [`LoginSessionMiddleware`](api:SilverStripe\SessionManager\Middleware\LoginSessionMiddleware)).
```

File: en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+#### Creating an extension for `LoginSession`
+
+The first step is to create a [`DataExtension`](api:SilverStripe\ORM\DataExtension) that grant some users the ability to hooks into [`LoginSession`](api:SilverStripe\SessionManager\Models\LoginSession)'s `canView()` and `canDelete()` methods. This example aligns the permissions on the [`LoginSession`](api:SilverStripe\SessionManager\Models\LoginSession) to the permission on the Member who owns the [`LoginSession`](api:SilverStripe\SessionManager\Models\LoginSession).
+
+Alternatively, you could call [`Permission::check()`](api:SilverStripe\Security\Permission::check()) to validate if the member has a predefined CMS permission. If you need even more granular permissions, you can implement a [`PermissionProvider`](https://docs.silverstripe.org/en/developer_guides/security/permissions/#permissionprovider) to define your own custom permissions.
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md]:
```suggestion
Alternatively, you could call [`Permission::check()`](api:SilverStripe\Security\Permission::check()) to validate if the member has a predefined CMS permission. If you need even more granular permissions, you can implement a [`PermissionProvider`](/developer_guides/security/permissions/#permissionprovider) to define your own custom permissions.
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
There's a few minor problems with links, but otherwise looks good.

Please target the `5.0` branch so this can take effect immediately.

File: en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md (hunk 2 above)
Comment (emteknetnz) [reply en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md]: Updated

File: en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md (hunk 3 above)
Comment (emteknetnz) [reply en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md]: Updated

File: en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md (hunk 1 above)
Comment (emteknetnz) [reply en/02_Developer_Guides/18_Cookies_And_Sessions/03_Managing_Sessions.md]: Updated

**Comment by emteknetnz** [comment]
closed in favour of https://github.com/silverstripe/developer-docs/pull/327

**Pull request closed without merging**